include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -527,6 +527,29 @@ typedef struct OrtApiBase OrtApiBase;
 */
 ORT_EXPORT const OrtApiBase* ORT_API_CALL OrtGetApiBase(void) NO_EXCEPTION;
 
+/** \brief Thread work loop function
+*
+* Onnxruntime will provide the working loop on custom thread creation
+* Argument is an onnxruntime built-in type which will be provided when thread pool calls OrtCustomCreateThreadFn
+*/
+typedef void (*OrtThreadWorkerFn)(void* ort_worker_fn_param);
+
+typedef const void* OrtCustomThreadHandle;
```

Comment:
That's untyped handle. Why not to make it typed? So that it would be impossible to use other pointer instead? It would require reinterpet_cast<OrtCustomThreadHandle>(), but it's cleaner than allowing to pass ANY pointer instead.

Reply:
An anonymous type added - "_OrtCustomHandleType".
And OrtCustomThreadHandle is defined to "const _OrtCustomHandleType*"